cluster/validate-cluster.sh

```diff
@@ -62,9 +62,6 @@ for (( i=0; i<${#MINION_NAMES[@]}; i++)); do
     if [[ "${count}" == "0" ]]; then
       echo -e "${color_red}Failed to find ${MINION_NAMES[$i]}, cluster is probably broken.${color_norm}"
       exit 1
-    else
-      echo -e " ${color_green}[working]${color_norm}"
-      break
```

Comment:
That is from the loop below (line 80), which wasn't changed in my recent PR. Before checking the kubelet installation, it would previously have printed 

Attempt 1 to find k8s-satnam-gke-node-1 ... [working]

and then later printed the note about checking the kubelet installation. See https://github.com/GoogleCloudPlatform/kubernetes/commit/df03cc3e0c94e4694c86fd827cb6b731555bc677